pkg/xray/resource/resource_xray_custom_issue.go

```diff
@@ -17,37 +17,28 @@ import (
 
 var validPackageTypes = []string{
 	"alpine",
-	"bower",
+	//"bower",
```

Comment:
Let's delete these commented out lines.

Reply:
We need to generate updated documentation. You can do this by:

```shell
make doc
```

Reply:
Is this list the same as the one for security policy? We may need 2 separate lists if they aren't the same.

Reply:
> Is this list the same as the one for security policy? We may need 2 separate lists if they aren't the same.

Lists should be the same according to product. 
Currently, the commented-out package types are for some reason not supported in v2/policies API (Product could not give me an answer as to why). We can include them for future support or remove them and create separate lists for both APIs. Also might consider moving validPackageTypes to another file if both APIs are using it.

Reply:
I think having 2 separate lists is best for now until we know for sure they are same list.